truffle/test/VaultNotaryTest.js

```diff
@@ -0,0 +1,384 @@
+const truffleAssert = require('truffle-assertions');
+const uuidv4 = require('uuid/v4');
+const uuidToHex = require('uuid-to-hex');
+
+const NotaryContract = artifacts.require("VaultNotary");
+
+
+
+function uuidToHex32(uuid) {
+    return uuid + '00000000000000000000000000000000'
+}
+
+
+contract('VaultNotary', async (accounts) => {
+    const OWNER = accounts[0];
+    const SHIPPER = accounts[1];
+    const CARRIER = accounts[2];
+    const MODERATOR = accounts[3];
+    const ATTACKER = accounts[9];
+
+    let contract;
+    before(async () =>{
+        contract = await NotaryContract.deployed();
+    });
+
+    async function registerVault() {
+        const vaultId = uuidToHex(uuidv4(), true);
+        const vaultUri = "uri";
+        const vaultHash = "hash";
+        await contract.registerVault(vaultId, vaultUri, vaultHash, {from: SHIPPER});
+        return vaultId;
+    }
+
+    //***************testing registerVault**************************************
+
+    it("should create a vault notary", async () => {
+        const vaultId = uuidToHex(uuidv4(), true);
+        const vaultUri = "uri";
+        const vaultHash = "hash";
+        const newVaultTx = await contract.registerVault(vaultId, vaultUri, vaultHash, {from: SHIPPER});
+
+        await truffleAssert.eventEmitted(newVaultTx, "VaultRegistered", ev => {
+            return ev.vaultId === uuidToHex32(vaultId) && ev.msgSender === SHIPPER;
+        });
+
+        await truffleAssert.eventEmitted(newVaultTx, "VaultUri", ev => {
+            return ev.vaultUri === "uri" && ev.vaultId == uuidToHex32(vaultId) && ev.msgSender === SHIPPER;
```

Comment:
```suggestion
            return ev.vaultUri === vaultUri && ev.vaultId == uuidToHex32(vaultId) && ev.msgSender === SHIPPER;
```

use the constants you defined at the top to DRY this up